docs/manual/java/guide/cluster/Cluster.md

```diff
@@ -42,23 +42,71 @@ In development you are typically only running the service on one cluster node. N
+Akka Cluster Bootstrap helps forming (or joining to) a cluster by using [Akka Discovery](https://doc.akka.io/docs/akka/2.5/discovery/index.html) to discover peer nodes. It is an alternative to configuring static seed-nodes in dynamic deployment environments such as on Kubernetes or AWS.
 
 It builds on the flexibility of Akka Discovery, leveraging a range of discovery mechanisms depending on the environment you want to run your cluster in.
 
-### Manual Cluster Formation
+Akka Cluster Bootstrap, in Lagom, can be disabled though property `lagom.cluster.bootstrap.enabled = false`. Note that this configuration flag has no effect if you declare seed-nodes explicitly in which case Akka Cluster Bootstrap won't be used.
+
+#### Akka Discovery
+
+In order to find the peer nodes and form a cluster, Akka Cluster Bootstrap need to be configured to use one of the existing Akka Discovery implementations.
+
+The snippet bellow explifies how to configure the Akka Cluster Boostrap to use the Akka Discovery Kubernetes API.
+
+```
+akka.management.cluster.bootstrap {
+  # example using kubernetes-api
+  contact-point-discovery {
+    discovery-method = kubernetes-api
+    service-name = "hello-lagom"
+  }
+}
+```
+Other existing implementations are: DNS, AWS, Consul, Marathon API and static Configuration. It's also possible to implement your own Akka Discovery implementation if needed.
+
+For more detailed and advanced configurations options, please consult the [Akka Cluster Bootstrap](https://developer.lightbend.com/docs/akka-management/1.0.0-RC2/bootstrap/) documentation and its [reference.conf](https://github.com/akka/akka-management/blob/v1.0.0-RC2/cluster-bootstrap/src/main/resources/reference.conf) file.
+
+
+#### Cluster Http Management
 
-If you prefer to not use **Akka Cluster Bootstrap** and handle the cluster formation yourself, you can disable it in your `application.conf` file and configure the Akka Cluster seed nodes statically.
+[Akka Cluster Bootstrap](https://developer.lightbend.com/docs/akka-management/1.0.0-RC2/bootstrap/) relies on [Http Cluster Management](https://developer.lightbend.com/docs/akka-management/1.0.0-RC2/cluster-http-management.html) to form a cluster.
 
-First, disable the cluster bootstrap:
+Akka Management Cluster HTTP is a [Akka Management](https://developer.lightbend.com/docs/akka-management/1.0.0-RC2/akka-management.html) extension that allows interaction with an akka-cluster through an HTTP interface. This management extension exposes different operations to manage nodes in a cluster (by default only read-only operations are exposed) as well as health checks based on the cluster state.
+
+Therefore, Akka Management will also be enabled and will run on http port `8558`. You can configured it to another port by setting property `akka.management.http.port` in your `application.conf` file.
```

Comment:
TIL: `therefore` is the correct spelling here ([instead of `therefor`](https://www.grammarly.com/blog/therefore-vs-therefor/) ) so nothing to change in this sentence since it is correct. 😅